src/authenticator.rs

```diff
@@ -157,6 +162,8 @@ pub struct AuthenticatorBuilder<C, F> {
 pub struct InstalledFlowAuthenticator;
 impl InstalledFlowAuthenticator {
     /// Use the builder pattern to create an Authenticator that uses the installed flow.
+    #[cfg(any(feature = "hyper-rustls", feature = "hyper-tls"))]
+    #[cfg_attr(yup_oauth2_docsrs, doc(cfg(any(feature = "hyper-rustls", feature = "hyper-tls"))))]
     pub fn builder(
```

Comment:
NB: with this PR there won't be a way to build any of the authenticators at the moment, we should add methods to start off with a custom client in the first place.

Reply:
But I would like to leave this up to a follow-up.

Reply:
> NB: with this PR there won't be a way to build any of the authenticators at the moment, we should add methods to start off with a custom client in the first place.

can you elaborate? I don't understand what you mean by this :)

Reply:
So, AFAIU right now if you want to build an authenticator, you must use one of these `builder` methods, which require a `DefaultHyperClient`. Since `DefaultHyperClient` does not exist, these methods can't either. And since this is the only way to build an authenticator, you are left without a way to construct one if you don't enable one of the features.

Does that clarify it?

Reply:
The ideal solution would be to allow user of this API to provide their own hyper client in the first place, rather than only having a way to override it from a default.

Reply:
I agree, not sure why this wasn't done earlier.